Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+# How to Configure Leap Motion Hand Tracking in MRTK 
+
+A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-controller/) is required to use this data provider.
+
+The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
+
+## Using Leap Motion tracking in MRTK
+1. Prepare MRTK project for Leap Motion
+
+    - This step only applies if the source of MRTK is cloned from the git repo and NOT the unity packages.
+
+        - Navigate to **Mixed Reality Toolkit > Utilities > Update > Configure CSC File for Leap Motion**. Updating the csc file filters out the obsolete warnings produced by the Leap Motion Core Assets.
+
+    ![LeapMotionUpdateCSCFile](../Images/CrossPlatform/LeapMotion/LeapMotionConfigureCSCFile.png)
+
+1. Importing the Leap Motion Core Assets
+    - Install the [Leap Motion SDK](https://developer.leapmotion.com/releases/?category=orion)
```

Comment:
Do we want to mention the specific version that these docs were written against? Otherwise we're somewhat vulnerable to an unexpected update (i.e. they get a new version and these instructions get broken).

Reply:
Yes, added the specific versions